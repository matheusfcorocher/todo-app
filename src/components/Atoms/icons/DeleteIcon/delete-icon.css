.delete-icon {
    filter: invert(27%) sepia(95%) saturate(7484%) hue-rotate(357deg) brightness(104%) contrast(115%);
}