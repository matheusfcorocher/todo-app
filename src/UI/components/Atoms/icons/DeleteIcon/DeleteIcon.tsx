import React, { useState } from 'react';
import '../../../../../App';
import './delete-icon.css';
import deleteIcon from '../../../../assets/icons/delete.svg';

function DeleteIcon() {
    return (
        <img 
            src={deleteIcon} 
            role="image" 
            className="delete-icon" 
            alt="delete" 
        />
    );
}

export default DeleteIcon;
