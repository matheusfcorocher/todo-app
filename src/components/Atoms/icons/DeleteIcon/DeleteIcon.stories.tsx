import React from 'react';
import { ComponentStory, ComponentMeta } from '@storybook/react';
import DeleteIcon from './DeleteIcon';

export default {
  title: 'Atoms/DeleteIcon',
  component: DeleteIcon,
  decorators: [
    (Story) => (
      <ul style={{ listStyleType: 'none', margin: 0, padding: 0 }}>
        <Story />
      </ul>
    ),
  ]
} as ComponentMeta<typeof DeleteIcon>;

const Template: ComponentStory<typeof DeleteIcon> = (args) => <DeleteIcon />;

export const Default = Template.bind({});
Default.args = {};

Default.args = {
  todosData: []
};