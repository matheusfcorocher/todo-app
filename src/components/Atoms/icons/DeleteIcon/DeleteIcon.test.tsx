import React, { useState } from 'react';
import { render, screen, cleanup } from '@testing-library/react';
import DeleteIcon from './DeleteIcon';

describe("::Components ::Atoms ::DeleteIcon", () => {
    test('renders icon of DeleteIcon', () => {
        render(<DeleteIcon />);
        const icon = screen.getByRole("image");
        expect(icon).toMatchSnapshot();
    })
})
