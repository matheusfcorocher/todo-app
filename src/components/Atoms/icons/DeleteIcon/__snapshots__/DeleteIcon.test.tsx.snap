// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`::Components ::Atoms ::DeleteIcon renders icon of DeleteIcon 1`] = `
<img
  alt="delete"
  class="delete-icon"
  role="image"
  src="delete.svg"
/>
`;
